recipes/libmysqlclient/all/conanfile.py

```diff
@@ -120,6 +120,7 @@ def package_info(self):
         self.cpp_info.libs = ["libmysql" if self.settings.os == "Windows" and self.options.shared else "mysqlclient"]
         self.cpp_info.names["cmake_find_package"] = "MySQL"
         self.cpp_info.names["cmake_find_package_multi"] = "MySQL"
```

Comment:
By the way, there is no official module or config file.

Reply:
[cmake](https://github.com/Kitware/VTK/blob/master/CMake/FindMySQL.cmake) is as close as official as we have I guess

Reply:
I just wanted to mention this, I don't want to remove those lines, it would be a breaking change in the recipe. FindMySQL is widely used, but not official (VTK is not CMake).

Reply:
indeed, thanks for the precision